packages/synthetix-main/contracts/mocks/AccountRBACMixinModuleMockStorage.sol

```diff
@@ -3,13 +3,14 @@ pragma solidity ^0.8.0;
 
 contract AccountRBACMixinModuleMockStorage {
     struct AccountRBACMixinModuleMockStore {
-        uint256 rbacValue;
+        uint256 stakeMock;
+        uint256 mintMock;
     }
 
     function _mixinModuleMockStore() internal pure returns (AccountRBACMixinModuleMockStore storage store) {
         assembly {
             // bytes32(uint(keccak256("io.synthetix.snx.accountrbacmixinmodule")) - 1)
-            store.slot := 0x1ce4ca68760c4591c093e69c4f3107e99b4faf6a64521759223b68428b166546
+            store.slot := 0x186e65a22afe779f65be55d2f02e866d66351197187020749997986235b258c1
```

Comment:
Bad hash?

@mjlescano 👀 
I recommend that we consider using this instead of hardcoded hashes:

```
contract SomeStorage {
  bytes32 private constant slot = bytes32(uint(keccak256("io.synthetix.snx.some_storage")) - 1);
  
function _someStore() internal pure returns (SomeStore storage store) {
  assembly {
    store.slot = slot;
  }  
}
```

Reply:
For me that's ok! But is going to be more expensive to deploy (probably not an issue).